app/controllers/api/v2/json_web_tokens_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V2
+    class JsonWebTokensController < V2::BaseController
+      include Api::Version2
+
+      before_action :find_user
+
+      api :POST, "/users/:id/json_web_tokens", N_("Generate JSON web token")
+      param :id, String, required: true
+      param :expires_at, DateTime, desc: N_("Expiration date of the token. If not set, token will never expire")
+      def generate
+        render json: { json_web_token: @user.jwt_token!(expiration: expires_at) }
+      end
+
+      api :DELETE, "/users/:id/json_web_tokens", N_("Invalidates all user's JSON web tokens")
+      param :id, String, required: true
+      def invalidate
```

Comment:
**nit** I prefer keeping the names to rest, just changing the user facing names (urls, action buttons) to them, so this would still be `destroy` and generate would be `create`.
But that is very much just my personal preference and me pointlessly nitpicking here :)